libandroid-navigation/src/main/java/com/mapbox/services/android/navigation/v5/navigation/MapboxNavigationNotification.java

```diff
@@ -89,14 +90,9 @@ private void initialize(Context applicationContext, MapboxNavigation mapboxNavig
     notificationManager = (NotificationManager) applicationContext.getSystemService(Context.NOTIFICATION_SERVICE);
     isTwentyFourHourFormat = DateFormat.is24HourFormat(applicationContext);
 
-    collapsedNotificationRemoteViews = new RemoteViews(applicationContext.getPackageName(),
-      R.layout.collapsed_navigation_notification_layout);
-    expandedNotificationRemoteViews = new RemoteViews(applicationContext.getPackageName(),
-      R.layout.expanded_navigation_notification_layout);
-
     pendingOpenIntent = createPendingOpenIntent(applicationContext);
-    PendingIntent pendingCloseIntent = createPendingCloseIntent(applicationContext);
-    expandedNotificationRemoteViews.setOnClickPendingIntent(R.id.endNavigationBtn, pendingCloseIntent);
+    pendingCloseIntent = createPendingCloseIntent(applicationContext);
+    buildRemoteViews();
```

Comment:
We're building the remote views on every notification update. Should this call be removed from here?